### DISCUSSION TITLE
Custom Metadata for nav_item

### DISCUSSION THREAD

**Discussion by tmeuze** [squidfunk/mkdocs-material#6057] [Q&A]
Hello, 

I have implemented method of displaying an overlay when you hover over navigation items. I've done this using the [custom metadata](https://squidfunk.github.io/mkdocs-material/reference/#customization) feature, with a partial override of `tab-items.html`. This overlay element includes an image as well as a summary, which is supposed to be pulled from the frontmatter of a page - but instead of `page.meta.summary`, I use `nav_item.meta.summary`, so that it always references the frontmatter in the page for the navigation item you are hovering, instead of the current page. 

Well, this worked just fine when the site was last successfully generated (v. 9.0.4 - insiders), but some time between then and now, this no longer works. I receive an error along the lines of:
```
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'summary'
```
I just want to know what has changed about the ability to use `nav_item.meta.*` - and if there is now a substitute template I should be using instead. 

Thank you for your time. 

Snippet of `tabs-item.html` override working in 9.0.4-insiders:

```
...
  <!-- Render item -->
  {% else %}
    <li class="md-tabs__item">
      <a href="{{ nav_item.url | url }}" class="{{ class }}">
        {{ title }}
      </a>
      <div class="cx-tab-item">
        <div class="cx-tab-item-desc">
          <div class="shadow"></div>
          <div class="cx-tab-item-contents">
            <div class="cx-tab-item-desc--text md-typeset">
              <p>{{ nav_item.meta.summary }}</p>
              <p>{% include "partials/bignav/" + nav_item.meta.bignav %}</p>
            </div>
            <div class="cx-tab-item-desc--image">
              <img src="/assets/images/headers/{{ nav_item.meta.headerimage }}" height="150px" width="150px"/>
            </div>
          </div>
        </div>
      </div>
    </li>
  {% endif %}
...
```

**Comment by squidfunk** [comment]
Thanks for asking. Have you checked if it works with the `mkdocs` theme? The fact that it doesn't work anymore might be related to a change in Material for MkDocs or MkDocs. AFAIK, there was no change in our code base preventing this.

**Reply by alexvoss** [reply]
The `meta` attribute of a [page object](https://www.mkdocs.org/dev-guide/themes/#page) should be there and if you have a `summary` in the page header that should be picked up, methinks. Perhaps you have a page where it is missing? Putting in a default value in such a case might help? 

The suggestion to try with the `mkdocs` theme sounds like a good approach to rule out any strange interactions with code in mkdocs-material.
